termux-instructions.md

```diff
@@ -0,0 +1,42 @@
+# fedoralovespython.org - Termux support
+Fedora Loves Python website
+
+# Problems
+- pip doesn't work
+- elsa not available as RPM
+
+# Instructions
+
+For running on an Android device:
+
+- 1. Install Termux and Termux-Fedora from https://github.com/nmilosev/termux-fedora
+- 2. Setup:
+```
+dnf update -y
+dnf install -y git python3-flask python3-markdown python3-yaml
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+git clone https://github.com/fedora-python/fedoralovespython.org
+cd fedoralovespython.org
+```
+- 3. You have to edit site.py (comment/remove these lines):
```

Comment:
Instructions to edit the source code should not be in documentation. Please change the source itself to make it work in all use cases.